### PR TITLE
Added things to the economy.json

### PR DESCRIPTION
**Pull request by jonatench**
All of them include me because yes 😄

### CODE REVIEW

**Comment by TechnoVisionDev** [comment]
Try to keep personalization out of economy responses. Replace "Realz" with a well known figure or character. For example "You break into Rapunzel's kingdom and steal her hair, you sell it on the black market for {amount}."

**Comment by jonatench** [comment]
Alright I've changed all of em to not include me

File: src/main/resources/localization/economy.json [hunk 1]
```diff
@@ -10,18 +10,21 @@
...
+    "Your NFT collection surges in price, so you sell them for {amount}.",
+    "You finished digging in Patrick's mines for the day, He pay's you {amount}."
   ],
   "crimeSuccess": [
     "You work for the Juarez cartel as a brick-presser and earn {amount}! They'll probably kill you and your family tomorrow. Just saying.",
     "You rob an orphanage for {amount}.",
     "You break into the house next door and sell their gaming PC on E-Bay for {amount}.",
-    "You work as a scam seller on Amazon selling fake airpods for full price and make {amount}."
+    "You work as a scam seller on Amazon selling fake airpods for full price and make {amount}.",
+    "You break into Rapunzels's kingdom and steal her hair, you sell it on the black market for {amount}."
```
Comment (TechnoVisionDev) [inline src/main/resources/localization/economy.json]: Grammar: "You break into Rapunzel's kingdom and steal her hair. You sell it on the black market for {amount}."

Alternate: "You break into Rapunzel's kingdom, steal her hair, and sell it on the black market for {amount}."

File: src/main/resources/localization/economy.json [hunk 2]
```diff
@@ -10,18 +10,21 @@
     "Your recent novel becomes a hit, earning you {amount} in royalties.",
     "You find a rare fossil on the beach and sell it to the museum for {amount}.",
     "You invested in the right stocks a the right time, earning {amount}.",
-    "Your NFT collection surges in price, so you sell them for {amount}."
+    "Your NFT collection surges in price, so you sell them for {amount}.",
+    "You finished digging in Patrick's mines for the day, He pay's you {amount}."
```
Comment (TechnoVisionDev) [inline src/main/resources/localization/economy.json]: Remove random names when referring to posession unless they are characters or public figures. "You finished digging in the mines for the day and make {amount}.

another alternative: "You finished digging in The Seven Dwarf's mine for the day. They pay you {amount}."

You get the idea

**Review by TechnoVisionDev — CHANGES_REQUESTED** [review]
I will merge the PR after these changes have been made

**Comment by jonatench** [comment]
alright there

**Pull request merged**